doc/changelogs/CHANGELOG_V18.md

```diff
@@ -34,6 +35,100 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="18.2.0"></a>
+
+## 2022-05-11, Version 18.2.0 (Current), @BethGriggs prepared by @RafaelGSS
```

Comment:
```suggestion
## 2022-05-17, Version 18.2.0 (Current), @BethGriggs prepared by @RafaelGSS
```

Reply:
(Also to be updated in the commit message, along with any notable change updates)